### ISSUE TITLE
Accuracy of the pretrained checkpoint:  SLDDBD_patchsize32_swin_ratio0.5_img224_statefarm_110ep

### ISSUE THREAD

**Issue by congduan-HNU** [Rocky1salady-killer/SL-DDBD#1]
We can't get the accuracy of 99.60% on the SFD dataset by your released 
 checkpoint. We inference on the all samples of origin SFD train subset, and get just about 30%. For the 'c8' folder, we just get 63.2%. Are there any problem of our config? We just run the 'inference.py' as your said.

**Comment by Rocky1salady-killer** [comment]
Hello Duan Cong! Thanks for your questions, here I simply answer, if you have any questions feel free to leave a message and send me an email!
We only open source two checkpoints at the moment. One is the imagenet-1k pre-trained checkpoint. The other is that the pre-trained model has done transfer learning on state-farm. This is not our best model of SL-DDBD.
Second, we used the official swin transformer test script main_eval.py. The inference script is something we added later and only implements simple inference. So I also recommend using main_eval.py to test the model. In addition, the sample of our test set is separated from the sample of the original training set in an 8:2 ratio. In other words, it may be unique, so there is a difference in test results. Because the SL-DDBD model uses an extended training set, we make a test set of 20,000 samples that is also extended by our proposed data enhancement strategy. The SL-DDBD model achieved 99.60 results on this test set of 20,000 samples.
I highly recommend that you use SwinMIM-Large checkpoints to re-run the transfer learning on the state farm dataset, and also change the encoder according to our lightweight strategy or new ideas of your own for different training and testing to get better results.

**Comment by wyhAndy** [comment]
usage: Swin Transformer training and evaluation script --cfg FILE
                                                       [--opts OPTS [OPTS ...]]
                                                       [--batch-size BATCH_SIZE]
                                                       [--data-path DATA_PATH]
                                                       [--pretrained PRETRAINED]
                                                       [--resume RESUME]
                                                       [--accumulation-steps ACCUMULATION_STEPS]
                                                       [--use-checkpoint]
                                                       [--amp-opt-level {O0,O1,O2}]
                                                       [--output PATH]
                                                       [--tag TAG] [--eval]
                                                       [--throughput]
                                                       --local_rank LOCAL_RANK
Swin Transformer training and evaluation script: error: the following arguments are required: --cfg, --local_rank
